lib/spack/docs/packaging_guide.rst

```diff
@@ -1898,30 +1903,30 @@ Inconsistent Specs
 
 Suppose a user needs to install package C, which depends on packages A
 and B.  Package A builds a library with a Python2 extension, and
-pacakge B builds a library with a Python3 extension.  Packages A and B
+package B builds a library with a Python3 extension.  Packages A and B
 cannot be loaded together in the same Python runtime:
 
-.. code-block:: none
+.. code-block:: python
 
     class A(Package):
         variant('python', default=True, 'enable python bindings')
         depends_on('python@2.7', when='+python')
-        def patch(self):
+        def install(self):
```

Comment:
Minor issue : if `install` is the right method here the signature is wrong, and should read : 
```python
def install(self, spec, prefix):
    ...
```